aggregating-profiler/src/main/java/ru/fix/aggregating/profiler/RegexpTagger.java

```diff
@@ -0,0 +1,39 @@
+package ru.fix.aggregating.profiler;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.HashMap;
+import java.util.regex.Pattern;
+
+/**
+ *
+ * @author Andrey Kiselev
+ */
+
+public class RegexpTagger implements Tagger {
+    private final String defaultTagName;
+    private final Map<String, Set<Pattern>> groupSeparator = new HashMap<>();
+
+    public RegexpTagger(String defaultTagName,
+                        Map<String, Set<Pattern>> groupSeparator) {
+        this.groupSeparator.putAll(groupSeparator);
+        this.defaultTagName = defaultTagName;
+    }
+
+    @Override
+    public <T extends Tagged> T assignTag(String profiledCallName,
+                                          T tagged) {
+        for(Map.Entry<String, Set<Pattern>> entry : groupSeparator.entrySet()) {
+             for(Pattern p : entry.getValue()) {
+                 if(p.matcher(profiledCallName).matches()) {
+                     tagged.setTag(defaultTagName, entry.getKey());
+                     return tagged;
+                 }
+             }
+         }
+        
+        tagged.setTag(defaultTagName, defaultTagName);
```

Comment:
It is not transparent, that RegexpTagger sets tag value same as tag name in case when the metric name does not match any pattern from configuration.   
What you think about other option:  in such case we can assign empty string value "". 
We can explicitly declare constant for that matter: 
interface Tagger{
  String EMPTY_VALUE = ""
}

Reply:
seems good. ok